newdle/client/package.json

```diff
@@ -3,11 +3,17 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "chai": "^4.2.0",
+    "enzyme": "^3.10.0",
```

Comment:
that's pretty certainly a dev dependency

Reply:
Yeah, indeed. I was to do it when cleaning up the code ;)